tox.ini

```diff
@@ -4,6 +4,10 @@ envlist=
     py35-core-stdlib
     py35-integration-{ethtestrpc,goethereum,ethtester}
     flake8
+    py36-ens
```

Comment:
Prefered format would be to inline these with the definitions above using the following syntax

```
py{35,36}-ens
py{35,36}-core  # you can drop the `-stdlib` suffix
py{35,36}-integration-{...}
```